nodes/srl/srl.go

```diff
@@ -247,11 +248,6 @@ func (s *srl) PreDeploy(_ context.Context, params *nodes.PreDeployParams) error
 func (s *srl) PostDeploy(ctx context.Context, params *nodes.PostDeployParams) error {
 	log.Infof("Running postdeploy actions for Nokia SR Linux '%s' node", s.Cfg.ShortName)
 
-	// populate config with mgmt ip addresses
-	if err := s.UpdateConfigWithRuntimeInfo(ctx); err != nil {
-		return err
-	}
-
```

Comment:
Hi @steiler, I removed this, since I think it is always performed for all nodes unconditionally in https://github.com/srl-labs/containerlab/blob/181b53924719af4a16f6781bd238de1b720a2557/cmd/deploy.go#L227

Reply:
ok, cool. Yeah did not specifically check to be honest :disappointed:.